docs/100_community/050_pepr-media.md

```diff
@@ -0,0 +1,24 @@
+# Pepr Media
+
+## 2024
+
+***Session Recordings***
```

Comment:
```suggestion
## Session Recordings
```
It's kinda weird if only one of the top-level headings _isn't_ a heading / is italic... no?